src/drivers/angular2.js

```diff
@@ -0,0 +1,99 @@
+import { omit, replaceObject } from '../lib';
+/* eslint-disable new-cap, object-shorthand */
+
+
+export let angular2 = {
+
+    global() {
```

Comment:
Potentially yeah. Especially if we can export a component directly and avoid requiring `BrowserModule`.